webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiConversationsSettings.js

```diff
@@ -84,6 +87,11 @@ export class SakaiConversationsSettings extends SakaiElement {
 
   render() {
 
+    // Use the existing guidelines content if available, otherwise use the sample
+    const guidelinesContent = (this.settings.guidelines && this.settings.guidelines.trim())
```

Comment:
You shouldn't need to do this here. You've done it already elsewhere. Just make guidelines a reactive property.